physionet-django/user/management/commands/reject_pending_credentialing_applications.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.db import transaction
+from django.http import HttpRequest
+from django.utils import timezone
+
+import notification.utility as notification
+from user.models import CredentialApplication, User
+
+LOGGER = logging.getLogger(__name__)
+REJECTION_MESSAGE = 'Your Reference {reference_name} did not respond to the reference check request within '\
+                    f'{settings.MAX_REFERENCE_VERIFICATION_DAYS_BEFORE_AUTO_REJECTION} days.'
+
+DFAULT_NUMBER_OF_APPLICATIONS_TO_REJECT = 5
```

Comment:
Typo: DFAULT_NUMBER_OF_APPLICATIONS_TO_REJECT -> DEFAULT_NUMBER_OF_APPLICATIONS_TO_REJECT

Reply:
I make typos like this all the time. You're too hard on yourself Amit!